cocos/spine/lib/spine-define.ts

```diff
@@ -106,486 +106,149 @@ function overrideClass (wasm): void {
-    spine.RegionAttachment = wasm.RegionAttachment;
-    spine.AtlasAttachmentLoader = wasm.AtlasAttachmentLoader;
-    spine.TextureAtlasPage = wasm.TextureAtlasPage;
-    spine.TextureAtlasRegion = wasm.TextureAtlasRegion;
-    spine.TextureAtlas = wasm.TextureAtlas;
-    spine.PowOut = wasm.PowOut;
-    spine.BoneData = wasm.BoneData;
-    spine.SlotData = wasm.SlotData;
-    spine.Updatable = wasm.Updatable;
-    spine.IkConstraint = wasm.IkConstraint;
-    spine.PathConstraint = wasm.PathConstraint;
-    spine.TransformConstraintData = wasm.TransformConstraintData;
-    spine.TransformConstraint = wasm.TransformConstraint;
-    spine.Bone = wasm.Bone;
-    spine.Slot = wasm.Slot;
-    spine.Skin = wasm.Skin;
-    spine.SkinEntry = wasm.SkinEntry;
-    spine.SkeletonClipping = wasm.SkeletonClipping;
-    spine.SkeletonData = wasm.SkeletonData;
-    spine.TranslateTimeline = wasm.TranslateTimeline;
-    spine.ScaleTimeline = wasm.ScaleTimeline;
-    spine.ShearTimeline = wasm.ShearTimeline;
-    spine.RotateTimeline = wasm.RotateTimeline;
-    spine.ColorTimeline = wasm.ColorTimeline;
-    spine.TwoColorTimeline = wasm.TwoColorTimeline;
-    spine.AttachmentTimeline = wasm.AttachmentTimeline;
-    spine.DeformTimeline = wasm.DeformTimeline;
-    spine.EventTimeline = wasm.EventTimeline;
-    spine.DrawOrderTimeline = wasm.DrawOrderTimeline;
-    spine.IkConstraintTimeline = wasm.IkConstraintTimeline;
-    spine.TransformConstraintTimeline = wasm.TransformConstraintTimeline;
-    spine.PathConstraintPositionTimeline = wasm.PathConstraintPositionTimeline;
-    spine.PathConstraintMixTimeline = wasm.PathConstraintMixTimeline;
-    spine.TrackEntry = wasm.TrackEntry;
-    spine.AnimationStateData = wasm.AnimationStateData;
-    spine.AnimationState = wasm.AnimationState;
-    spine.Animation = wasm.Animation;
-    spine.EventQueue = wasm.EventQueue;
-    //spine.AnimationStateListener = wasm.AnimationStateListener;
-    spine.AnimationStateAdapter = wasm.AnimationStateAdapter;
-    spine.Skeleton = wasm.Skeleton;
-    spine.SkeletonBinary = wasm.SkeletonBinary;
-    spine.SkeletonJson = wasm.SkeletonJson;
-    spine.VertexEffect = wasm.VertexEffect;
-    spine.JitterEffect = wasm.JitterEffect;
-    spine.SwirlEffect = wasm.SwirlEffect;
-
-    spine.SkeletonInstance = wasm.SkeletonInstance;
-    spine.SkeletonSystem = wasm.SkeletonSystem;
+    Object.assign(spine, wasm);
 }
 
 function overrideProperty_String (): void {
     const prototype = spine.String.prototype as any;
     const propertyPolyfills = [
-        {
-            proto: prototype,
-            property: 'length',
-            getter: prototype.length,
-        },
-        {
-            proto: prototype,
-            property: 'isEmpty',
-            getter: prototype.isEmpty,
-        },
-        {
-            proto: prototype,
-            property: 'str',
-            getter: prototype.str,
-        },
+        ['length', prototype.length],
```

Comment:
Why can change object to array?

Reply:
For smaller code.  You could compare the code, and using array will make the code more clean.